openmdao/core/system.py

```diff
@@ -1366,7 +1384,7 @@ def list_states(self, stream=sys.stdout):
         unknowns = self.unknowns
         resids = self.resids
         states = []
-        for uname in unknowns:
+        for uname in iterkeys(unknowns):
```

Comment:
do we need iterkeys here since iterating over a dict already gives you the keys?